Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1883,10 +1883,38 @@ extension BrowserViewController: TabManagerDelegate {
         tab.appStateDelegate = self
     }
 
+    func addTabToRecentTabs(tab: Tab) {
+        if var recentTabs = profile.prefs.dictionaryForKey("recentlyClosedTabs") {
+            var urls = recentTabs["urls"] as! [String]
```

Comment:
Nit: I think using a `guard` statement here and combining the `as!` with the previous unwrap would make this cleaner/safer

```swift
guard var recentTabs = profile.prefs.dictionaryForKey("recentlyClosedTabs"), 
      var urls = recentTabs["urls"] as? [String] else {
    // Do else clause from below here
}
```